conf/ux/expectations.conf.schema.json

```diff
@@ -0,0 +1,220 @@
+{
+    "$schema": "http://json-schema.org/draft/2020-12/schema",
+    "title": "Expectations and Notifications",
+    "description": "Specifies which trips users are expected to label and when they should be reminded to do so",
+    "type": "object",
+    "properties": {
+        "modes": {
+            "description": "The various modes we can be in and their specifications. The active mode is the first mode in this list with a schedule matching the current date or with no schedule.",
+            "type": "array",
+            "items": {
+                "$ref": "#/definitions/modeType"
+            },
+            "contains": {
+                "$comment": "Must contain a \"default\" element -- one with no schedule",
+                "not": {"required": ["schedule"]}
+            }
+        }
+    },
+    "required": ["modes"],
+    "definitions": {
+        "modeType": {
+            "description": "Specifies a collection mode that defines expectation and notification settings for a given time period",
+            "type": "object",
+            "properties": {
+                "enabled": {
+                    "description": "Whether or not to enable this mode; mode is enabled if this property is omitted",
+                    "type": "boolean"
+                },
+                "label": {
+                    "description": "A human-readable label for the mode",
+                    "type": "string"
+                },
+                "confidenceThreshold": {
+                    "description": "Only display yellow labels with confidence greater than this threshold (0 to display all, 1 to display none)",
```

Comment:
This is actually something I haven't figured out yet. If the confidence for a given label is below the threshold, the phone should always display that label as red and blank; that part I'm sure of (and have implemented accordingly on the client side). The current implementation has these below-threshold labels also counting as red for the purposes of rule matching as well, but I'm not sure what the best choice is there.

If we wanted to be maximally flexible, we would _not_ have below-threshold labels match `"trigger": -1` because there is a semantic difference between very low confidence inferences and no inferences at all, and if a study admin wanted to treat them the same they could just write one more rule to explicitly match these trips. The argument in favor of having below-threshold labels match `"trigger": -1` (the status quo) is that it forces expectation/notification behavior to match what the user actually sees: to the user, red labels are red labels, if some trips that display red labels are appearing in `To Label` and some aren't (and it's not because of some random sample option), they might get confused. It's also simpler.

Reply:
I agree that this is complex. Let's discuss it after tomorrow. I really want to deploy this today so we can walk through the new screen at tomorrow's deployer/beta tester meeting.